packages/rich-text/src/plugins/Hyperlink/components/LinkPopover.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 
 import { Popover, IconButton, Tooltip, Flex } from '@contentful/f36-components';
 import { EditIcon, CopyIcon } from '@contentful/f36-icons';
+import * as Slate from 'slate-react';
```

Comment:
see import from previously PR
I think we want to import slate and plate always from internal to make updates easier